modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/AccountResourceTest.java

```diff
@@ -987,6 +991,61 @@ private void _testPatchAccountWithContactInformation() throws Exception {
 			patchAccount.getAccountContactInformation());
 	}
 
+	private void _testPatchAccountWithEmptyOrganizationExternalReferenceCodeArray()
+		throws Exception {
+
+		Account randomAccount = randomAccount();
+
+		Organization organization1 = OrganizationTestUtil.addOrganization();
+
+		Organization organization2 = OrganizationTestUtil.addOrganization();
+
+		randomAccount.setOrganizationExternalReferenceCodes(
+			new String[] {
+				organization1.getExternalReferenceCode(),
+				organization2.getExternalReferenceCode()
+			});
+
+		Account postAccount = _postAccount(randomAccount);
+
+		postAccount.setOrganizationExternalReferenceCodes(new String[0]);
+		postAccount.setOrganizationIds(new Long[0]);
+
+		Account patchAccount = accountResource.patchAccount(
+			postAccount.getId(), postAccount);
```

Comment:
It's a little strange to me here, that you use the same postAccount object for patching. Usually in the testPatch... methods, we use a randomPatchAccount, with all new values, like in _testPatchAccountWithMoreExternalReferenceCodes. 

But if this was already discussed with @smottal  and it is fine like this, please disregard this comment

Reply:
for me it's fine